.github/workflows/full_suite_integration_tests.yml

```
@@ -157,7 +157,7 @@ jobs:
         run: |
           flankScripts integration_tests process_results \
           --global-result=${{ needs.run-it-full-suite.outputs.job_status }} \
-          --run-result=${{ toJSON(env) }} \
+          --run-result='${{ toJSON(env) }}' \
```

hmmm 
i was just about to commit 
```suggestion
          --run-result=\"${{ toJSON(env) }}\" \
```